frontend/src/scenes/settings/project/PersonsOnEvents.tsx

```diff
@@ -0,0 +1,74 @@
+import { useActions, useValues } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonRadio, LemonRadioOption } from 'lib/lemon-ui/LemonRadio'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+
+import { HogQLQueryModifiers } from '~/queries/schema'
+
+type PoEMode = NonNullable<HogQLQueryModifiers['personsOnEventsMode']>
+
+const poeOptions: LemonRadioOption<PoEMode>[] = [
+    {
+        value: 'person_id_no_override_properties_on_events',
+        label: 'Deprecated old PoE setting. May incorrectly aggregate users. You probably want one of the other options.',
+    },
+    {
+        value: 'person_id_override_properties_on_events',
+        label: 'Use ingestion-time person properties from the events table (faster)',
+    },
+    {
+        value: 'person_id_override_properties_joined',
+        label: 'Use current person properties from the persons table (slower)',
+    },
```

Comment:
Proposal for wording: https://posthog.slack.com/archives/C0368RPHLQH/p1714689440221919?thread_ts=1714679592.721539&cid=C0368RPHLQH

- Deprecated: Use person ids and properties from the time of the event. May show higher unique user counts due to not using latest person ids. You probably want one of the other options.
- Use person properties from the time of the event. Faster queries. If person property is updated, then query results on past data won't change.
- Use latest person properties. Slower queries. If person property is updated, then query results on past data will change to reflect it.